runtime/fundamentals/ts_support.md

```diff
@@ -0,0 +1,292 @@
+---
+title: "TypeScript support"
+oldUrl:
+  - /runtime/manual/advanced/typescript/
+  - /runtime/manual/typescript/
+  - /runtime/manual/typescript/overview/
+  - /runtime/manual/getting_started/typescript/
+  - /manual/advanced/typescript
+  - /runtime/manual/advanced/typescript/overview/
+  - /runtime/manual/advanced/typescript/types/
+---
+
+Deno treats TypeScript as a first class language, just like JavaScript or
```

Comment:
"treats" sounds odd here. Better: "In Deno, TypeScript is a first class language..."